src/types/models.types.ts

```diff
@@ -0,0 +1,43 @@
+export class Judgments {
+  fantasticPlus: number;
+  fantastics: number;
+  excellents: number;
+  greats: number;
+  decents?: number;
+  wayOffs?: number;
+  misses: number;
```

Comment:
You'll probably want to add holds, rolls, mines etc. here as well

Reply:
Done.